src/managing-workflow/aws-eks-running-builds-on-fargate.md

```diff
@@ -0,0 +1,247 @@
+  name: deis-builder-fargate
+  namespace: deis-builder
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: deis-builder-fargate
+subjects:
+- kind: ServiceAccount
+  name: deis-builder
+  namespace: deis
+```
+
+Save the file and then apply the RBAC authorization rules using the following command:
+
+```shell
+kubectl apply -f rbac.yaml
+```
+
+This will grant the necessary permissions to the `deis-builder` service account in the `deis-builder` namespace.
+
+## Duplicating Secrets in the New Namespace
+
+To make the secrets available in the `deis-builder` namespace, you need to duplicate them from the default `deis` namespace. Follow these steps:
+
+1. Create a file named `secrets.yaml` and add the following content to it:
+
+```yaml
+apiVersion: v1
+data:
+  builder-key: TO_REPLACE
+kind: Secret
+metadata:
+  name: builder-key-auth
+  namespace: deis-builder
+type: Opaque
+---
+apiVersion: v1
+data:
+  ssh-host-ecdsa-key: TO_REPLACE
+  ssh-host-rsa-key: TO_REPLACE
+kind: Secret
+metadata:
+  name: builder-ssh-private-keys
+  namespace: deis-builder
+type: Opaque
+---
+apiVersion: v1
+data:
+ accesskey: TO_REPLACE
+ builder-bucket: TO_REPLACE
+ database-bucket: TO_REPLACE
+ region: TO_REPLACE
+ registry-bucket: TO_REPLACE
+ secretkey: TO_REPLACE
+kind: Secret
+metadata:
+ annotations:
+   deis.io/objectstorage: s3
+ name: objectstorage-keyfile
+ namespace: deis-builder
+type: Opaque
+```
+
+2. Replace the `TO_REPLACE` placeholders in the file with the actual values of the secrets. You can find the secrets in the default `deis` namespace using the following command:
+
+```shell
+kubectl get secret builder-key-auth -n deis -o jsonpath='{.data}'
+```
+
+3. Apply the secrets to the new namespace using the following command:
+
+```shell
+kubectl apply -f secrets.yaml
+```
+
+This will create the secrets in the `deis-builder` namespace, making them available for use by the `slugbuild` pods.
+
+## Setting Up Capacity for the `slugbuild` Pod
+
+By default, Fargate runs `slugbuild` pods with the lowest available capacities for CPU and memory. To change this behavior, you can set a default capacity for all pods spawned into the `deis-builder` namespace by creating a limit range.
+
+Follow these steps to set up the capacity for the `slugbuild` pod:
+
+1. Create a file named `limit-range.yaml` and add the following content to it:
+
+```yaml
+apiVersion: v1
+kind: LimitRange
+metadata:
+  name: deis-builder-defaults
+  namespace: deis-builder
+spec:
+  limits:
+  - default:
+      defaultRequest:
+        memory: 32Gi
+        cpu: 16
+    type: Container
+```
+
+2. Adjust the `defaultRequest` values (`memory` and `cpu`) according to your requirements.
+
+3. Apply the limit range to the `deis-builder` namespace using the following command:
+
+```shell
+kubectl apply -f limit-range.yaml
+```
+
+This will set the default capacity for all containers in the `deis-builder` namespace, including the `slugbuild` pods.
+
+
+## Cleanup operator
```

Comment:
@robinmonjo , thank you for the guide and contribution .

Looks like there is a duplicate section about the cleanup-operator here? Seems like lines 209-226 are equivalent to the lines after those?
https://github.com/teamhephy/workflow/pull/151/files#diff-636490027cfc34ec7049a941bd713f01a0edbbe9923f97bbe56795af05aa3160R209-R226

Reply:
Yes sorry, I did not remove the initial draft version of this part. Good now 😊